File: app/src/main/java/com/nacoda/moviesmvvm/base/BaseActivity.java
package com.nacoda.moviesmvvm.base;

import android.os.Bundle;
import android.os.PersistableBundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

/**
 * Created by Mayburger on 12/22/17.
 */

public class BaseActivity extends AppCompatActivity {

    AppCompatActivity mActivity;

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState, @Nullable PersistableBundle persistentState) {
        super.onCreate(savedInstanceState, persistentState);
        mActivity = this;
    }
}

File: app/src/main/java/com/nacoda/moviesmvvm/base/BaseActivity.kt
package com.nacoda.moviesmvvm.base

import android.os.Bundle
import android.os.PersistableBundle
import android.support.v7.app.AppCompatActivity

/**
 * Created by Mayburger on 12/22/17.
 */

open class BaseActivity : AppCompatActivity() {

    internal lateinit var mActivity: AppCompatActivity

    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)
        mActivity = this
    }
}
